ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -407,8 +407,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-minimal-permission
-- as: aws-ipi-ovn-efs-fips-f14-ui
-  cron: 48 4 1,16 * *
+- as: aws-ipi-ovn-efs-fips-f7-ui
+  cron: 0 0 * * 6
```

Comment:
Use the shell [script](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh) to generate the cron entry, so that all jobs are spread across the day/week/month.